index.js

```diff
@@ -104,6 +104,11 @@ function FSWatcher(_opts) {
       opts.usePolling = !!envLower
     }
   }
+  var envInterval = process.env.CHOKIDAR_INTERVAL;
+  if (envInterval !== undefined) {
```

Comment:
Simply `if (envInterval)` should be fine in this case, since we're not contending with `false` as a valid potential value.